docs/data/material/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -55,68 +119,283 @@ export default function Checkout() {
+          lg={8}
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            maxWidth: '100%',
+            width: '100%',
+            backgroundColor: { xs: 'transparent', sm: 'background.default' },
+            alignItems: 'start',
+            pt: { xs: 2, sm: 4 },
+            px: { xs: 2, sm: 10 },
+            gap: { xs: 4, md: 8 },
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: { sm: 'space-between', md: 'flex-end' },
+              alignItems: 'center',
+              width: '100%',
+              maxWidth: { sm: '100%', md: 600 },
+            }}
+          >
+            <Box
+              sx={{
+                display: { xs: 'flex', md: 'none' },
+                flexDirection: 'row',
+                width: '100%',
+                justifyContent: 'space-between',
+              }}
+            >
+              <Button
+                startIcon={<ArrowBackRoundedIcon />}
+                component="a"
+                href="/material-ui/getting-started/templates/landing-page/"
+                sx={{ alignSelf: 'start' }}
+              >
+                Back to
+                <img
+                  src={
+                    'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                  }
+                  style={logoStyle}
+                  alt="Sitemark's logo"
+                />
+              </Button>
+              <ToggleColorMode mode={mode} toggleColorMode={toggleColorMode} />
+            </Box>
+            <Box
+              sx={{
+                display: { xs: 'none', md: 'flex' },
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+                alignItems: 'flex-end',
+                flexGrow: 1,
+                height: 150,
+              }}
+            >
+              <ToggleColorMode mode={mode} toggleColorMode={toggleColorMode} />
+              <Stepper
+                id="desktop-stepper"
+                activeStep={activeStep}
+                sx={{
+                  width: '100%',
+                  height: '40px',
+                }}
+              >
+                {steps.map((label) => (
+                  <Step
+                    sx={{
+                      ':first-child': { pl: 0 },
+                      ':last-child': { pr: 0 },
+                    }}
+                    key={label}
+                  >
+                    <StepLabel>{label}</StepLabel>
+                  </Step>
+                ))}
+              </Stepper>
+            </Box>
+          </Box>
+          <Card
+            sx={{
+              display: { xs: 'flex', md: 'none' },
+              width: '100%',
+            }}
+          >
+            <CardContent
+              sx={{
+                display: 'flex',
+                width: '100%',
+                alignItems: 'center',
```

Comment:
```suggestion
                alignItems: 'center',
                justifyContent: 'space-between',
```

Using this instead of flex-grow below